packages/components/molecules/f-alert/package.json

```diff
@@ -52,7 +52,7 @@
     "@vue/test-utils": "1.0.3",
     "@justeat/f-wdio-utils": "0.3.0",
     "node-sass-magic-importer": "5.3.2",
-    "@justeat/f-button": "2.0.0-beta.0",
+    "@justeat/f-button": "2.0.0",
```

Comment:
I think you'll also need to bump the version on each of these packages so that they can be deployed to npm?

Reply:
Ohh, sure, I pushed to early